tools/coqdep.ml

```diff
@@ -525,17 +144,8 @@ let coqdep () =
   (* Add current dir with empty logical path if not set by options above. *)
   (try ignore (Coqdep_common.find_dir_logpath (Sys.getcwd()))
    with Not_found -> add_norec_dir_import add_known "." []);
-  (* NOTE: These directories are searched from last to first *)
   if !option_boot then begin
-    add_rec_dir_import add_known "theories" ["Coq"];
-    add_rec_dir_import add_known "plugins" ["Coq"];
-    add_rec_dir_import (fun _ -> add_caml_known) "theories" ["Coq"];
-    add_rec_dir_import (fun _ -> add_caml_known) "plugins" ["Coq"];
-    let user = "user-contrib" in
-    if Sys.file_exists user then begin
-      add_rec_dir_no_import add_known user [];
-      add_rec_dir_no_import (fun _ -> add_caml_known) user [];
-    end;
+    ()
```

Comment:
We may turn this conditional into `if not !option_boot then begin ... end;` (no else branch)

Reply:
Updated, thanks.